site/src/component/SearchModule/SearchModule.scss

```diff
@@ -12,6 +12,10 @@
     text-align: center;
     justify-content: center;
   }
+
+  .form-group {
+    margin-bottom: 0;
+  }
```

Comment:
this also relies on bootstrap, which is being removed from the search module in #848